packages/editor/src/components/post-featured-image/index.js

```diff
@@ -105,7 +100,7 @@ function PostFeaturedImage( {
 	const mediaUpload = useSelect( ( select ) => {
 		return select( blockEditorStore ).getSettings().mediaUpload;
 	}, [] );
-	const postLabel = get( postType, [ 'labels' ], {} );
```

Comment:
Instead of defaulting to `{}`, you could write `postLabels?.featured_image` at the places where the object is used. It's more consistent with how `postType?.labels?.whatever` is used at other places, many of which are modified in this PR.

Reply:
Sure, done as part of bfdd8424e431add15e6300d600942a1173aaaf69